src/version.h

```diff
@@ -12,7 +12,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 71000;
+static const int PROTOCOL_VERSION = 71400; // TODO (BDAP): ***** Revert to 71000
```

Comment:
71000

Reply:
That version is needed for our testnet but I have this commit to revert the version to 2.4.0 after this merge:  https://github.com/duality-solutions/Dynamic/commit/717f18d361993ecfc17415f4c8aaef2cd61c3793